### ISSUE TITLE
Update out-of-the-box version in VS

### ISSUE THREAD

**Issue by luisquintanilla** [dotnet/machinelearning-modelbuilder#2738]
The current version that comes with VS when you install it for the first time is 16.3.9.2235601.

Upgrade to the latest version of Model Builder.

**Comment by LittleLittleCloud** [comment]
The June release has a perf-regression issue caused by a binding redirect package which also shipped by VS core.

The binding-redirect package is required by `Azure.SDK`, where the version shipped in VS core is older than the version required by model builder

There're two solutions to eliminate that binding redirection
- move `Azure.SDK` into `AutoMLService`, so model builder wizard doesn't need to depend on the latest `Azure.SDK`
- update `Azure.SDK` in VS core.

After discussing with @JakeRadMSFT, we both support the first solution as the second solution will require model builder to target the latest release of VS.

In summary, here's the action item
- update insertion using Apirl release, that's the latest release before introducing binding redirect to Azure.core dependencies
- move all azure call into automlservice
